numpy/typing/tests/test_typing.py

```diff
@@ -48,8 +48,11 @@ def run_mypy() -> None:
 
     The mypy results are cached in `OUTPUT_MYPY` for further use.
 
+    The cache refresh can be skipped using
+
+    NUMPY_TYPING_TEST_CLEAR_CACHE=0 pytest numpy/typing/tests
     """
-    if os.path.isdir(CACHE_DIR):
+    if os.path.isdir(CACHE_DIR) and bool(os.environ.get("NUMPY_TYPING_TEST_CLEAR_CACHE", True)):
```

Comment:
@BvB93 I added this to speed up pytest runs. Seems pretty harmless but wanted to draw your attention to it.

Reply:
This should be fine I suppose, just be aware that a stale mypy cache can potentially causes unexpected results.